src/as-release.c

```diff
@@ -212,6 +212,9 @@ AsReleaseKind
 as_release_get_kind (AsRelease *release)
 {
 	AsReleasePrivate *priv = GET_PRIVATE (release);
+
```

Comment:
nit: For these one-line return/set functions with just one obvious check, I think you can just drop the two empty lines, they just add visual noise.